user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -21,13 +16,96 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
+
+1. Make sure you have at least **DataMiner CloudGateway 2.8.0** installed on the system. See [Upgrading nodes to the latest DxM versions](xref:Managing_cloud-connected_nodes#upgrading-nodes-to-the-latest-dxm-versions).
+
+1. Contact your Skyline representative or <staas@dataminer.services> to register you system to use STaaS.
+
+1. Wait until the registration is completed.
+
+1. **Optionally**, contact your Skyline representative or <staas@dataminer.services> to migrate your existing data to STaaS.
+
+1. In the `C:\Skyline DataMiner` folder, open *DB.xml* and edit it to look like this:
+
+   ```xml
+   <?xml version="1.0"?>
+   <DataBases xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://www.skyline.be/config/db">
+      <DataBase active="true" local="true" search="true" cloud="true" type="CloudStorage"/>
+   </DataBases>
+   ```
+
+1. Restart your DMA and STaaS will be used.
+
+## Limitations
+
+To **migrate existing data** to STaaS, the following limitations apply:
+
+- If you migrate from a Cassandra database, logger tables cannot be migrated yet. Support for the migration of logger tables will be introduced in a future DataMiner version.
+- Migration of an Elasticsearch database that is defined in [DBConfiguration.xml](xref:DBConfiguration_xml) instead of [DB.xml](xref:DB_xml) (e.g. because a setup with multiple Elasticsearch clusters is used) is not yet supported.
+- Migration from a MySQL setup is not yet supported.
+
+In addition, the following **other limitations** currently apply:
+
+- [Jobs](xref:jobs), [Ticketing](xref:ticketing), and [API Deployment](xref:Overview_of_Soft_Launch_Options#apideployment) data are not supported.
+- Indexing engine functionality such as search, suggestions, aliases, and aggregation are not supported.
+- Direct queries from DataMiner Cube to the database are not supported.
+- The [SLReset tool](xref:Factory_reset_tool) is not supported.
+- [Exporting trend data](xref:Exporting_elements_services_etc_to_a_dmimport_file) to a .dmimport file is not supported.
+- Proxy and DMZ setups are currently not supported.
+- The [autoincrement](xref:Protocol.Params.Param.ArrayOptions.ColumnOption-type#autoincrement) tag on logger tables is not supported.
+- DOM queries can be slower depending on the number of DOM records and the complexity of the query. This limitation will be removed in the near future.
+
+## Troubleshooting
+
+> [!NOTE]
+> If you experience any issues during setup or while using Storage as a Service, and you cannot resolve these with the procedures below, contact <staas@dataminer.services>.
+
+### Cloud connection lost
+
+Under normal circumstances, CloudGateway refreshes the cloud session automatically. However, if CloudGateway is down for longer than three days, for example because the server is down, the cloud session will become invalid. This will cause DataMiner startup to fail.
+
+To resolve this issue, use the following workaround:
```

Comment:
Is it sufficient to do this for one DMA to get an entire DMS up and running again? Or do you need to do it for every DMA?

Reply:
the token only needs to be refreshed once, so doing it from one DMA is enough